### PR TITLE
Update and rename build-processing.md to Pipelines.md

### PR DESCRIPTION
**Pull request by thomasr8**
Replaced "build processing" with "pipelines." Also made a few minor changes to the page.

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/Pipelines.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (michelle-luna) [inline jekyll/_cci2/Pipelines.md]: sorry, we can't handle renames, so this filename should be changed back to build-processing.md

File: jekyll/_cci2/Pipelines.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+layout: classic-docs
+title: "Pipelines"
+short-title: "Enabling Pipelines"
+description: "How to enable the preview of pipelines"
+categories: [settings]
+order: 1
+---
+
+This document describes how to enable the preview pipelines engine if you need to trigger workflows from the CircleCI API or auto-cancel workflows.
```
Comment (michelle-luna) [inline jekyll/_cci2/Pipelines.md]: remove the word "preview"

File: jekyll/_cci2/Pipelines.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+- Anchors will be processed and resolved instead of appearing in the app config.
+- If you use `<<` in your shell commands (most commonly found in use of heredocs) you will need to escape them using backslash `\` as in `\<<` in order to use version 2.1 or higher of configuration.
+- Pipelines are **not** fully backwards-compatible with the 1.1 API endpoint to trigger arbitrary jobs - you may experience unexpected or inconsistent results if you use this endpoint after turning on Pipelines. Alternatively, you can use the [build-triggering endpoint in the 1.1 API](https://circleci.com/docs/api/v1-reference/#new-project-build) introduced in September 2018. Please keep in mind that this build triggering API endpoint does **not** accept parameters or workflow or job filters. If you make heavy use of those API features and want to use Pipelines, please contact your CircleCI account team.
+
+## Giving Feedback
+1. Come to [CircleCI Discuss](https://discuss.circleci.com/t/2-1-config-and-build-processing/24102) to post feedback.
```
Comment (michelle-luna) [inline jekyll/_cci2/Pipelines.md]: remove bullet #1, this thread has been closed

**Comment by michelle-luna** [comment]
oh yeah, we also want to update the other files with build processing in them and add pipelines to the glossary

File: jekyll/_cci2/Pipelines.md (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/Pipelines.md]: Renamed doc back to original "Build-Processing.md"

File: jekyll/_cci2/Pipelines.md (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/Pipelines.md]: Removed

File: jekyll/_cci2/Pipelines.md (hunk 3 above)
Comment (thomasr8) [reply jekyll/_cci2/Pipelines.md]: Removed bullet #1 from content.

**Pull request merged**